src/group_theory/solvable.lean

```diff
@@ -227,12 +227,12 @@ begin
   ... = ⊥ : rfl,
 end
 
-lemma is_solvable_of_top_eq_bot (h : (⊤ : subgroup G) = ⊥) : is_solvable G :=
+instance is_solvable_of_top_eq_bot (h : (⊤ : subgroup G) = ⊥) : is_solvable G :=
 ⟨⟨0, by simp *⟩⟩
 
 variables {G} {G' : Type*} [group G'] {f : G →* G'}
 
-lemma solvable_of_solvable_injective (hf : function.injective f) (h : is_solvable G') :
+instance solvable_of_solvable_injective (hf : function.injective f) [h : is_solvable G'] :
```

Comment:
Sorry, this one shouldn't be an `instance`, because the type class system can't figure out `hf`. But you can still use `[]` for `h`.

Reply:
Oops, should be fixed now.